README.md

```diff
@@ -85,932 +119,667 @@ Unlike many authorization systems where a service controls access to resources i
+The data contained in this map MUST NOT be semantically meaningful to delegation chains.
 
-An agent executing a capability MUST verify that the outermost `aud` field _matches its own DID._ The associated ability MUST NOT be performed if they do not match. Recipient validation is REQUIRED to prevent the misuse of UCANs in an unintended context.
-
-The following UCAN fragment would be valid to invoke as `did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV`. Any other agent MUST NOT accept this UCAN. For example, `did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp` MUST NOT run the ability associated with that capability.
+Below is an example:
 
 ``` js
 {
-  "aud": "did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV",
-  "iss": "did:key:zAKJP3f7BD6W4iWEQ9jwndVTCBq8ua2Utt8EEjJ6Vxsf",
   // ...
+  "meta": {
+    "challenges": {
+      "example.com": "abcdef",
+      "another.example.net": "12345"
+    },
+    "sha3_256": {
+      "B94D27B9934D3E08A52E52D7DA7DABFAC484EFE37A5380EE9088F7ACE2EFCDE9": "hello world"
+    }
+  }
 }
 ```
 
-A good litmus test for invocation validity by a discharging agent is to check if they would be able to create a valid delegation for that capability.
+# Canonicalization
 
-### 6.2.2 Token Uniqueness
+## Cryptosuite
 
-Each remote invocation MUST be a unique UCAN: for instance using a nonce (`nnc`) or simply a unique expiry. The recipient MUST validate that they have not received the top-level UCAN before. For implementation recommentations, please refer to the [replay attack prevention] section. 
+Across all UCAN specifications, the following cryptosuite MUST be supported:
 
-## 6.3 Proof Chaining
+| Role      | REQUIRED Algorithms               | Notes                                | 
+|-----------|-----------------------------------|--------------------------------------|
+| Hash      | [SHA-256]                         |                                      |
+| Signature | [Ed25519], [P-256], [`secp256k1`] | Preference of Ed25519 is RECOMMENDED |
+| [DID]     | [`did:key`]                       |                                      |
 
-Each capability MUST either be originated by the issuer (root capability, or "parenthood") or have one-or-more proofs in the `prf` field to attest that this issuer is authorized to use that capability ("introduction"). In the introduction case, this check MUST be recursively applied to its proofs until a root proof is found (i.e. issued by the resource owner).
+## Encoding
 
-Except for rights amplification (below), each capability delegation MUST have equal or narrower capabilities from its proofs. The time bounds MUST also be equal to or contained inside the time bounds of the proof's time bounds. This lowering of rights at each delegation is called "attenuation."
+All UCANs MUST be canonically encoded with [DAG-CBOR] for signing. A UCAN MAY be presented or stored in other [IPLD] formats (such as [DAG-JSON]), but converted to DAG-CBOR for signature validation.
```

Comment:
I feel like the ` { "/" : { "bytes": "...` notation needs to be explained a little here, rather than assuming the reader both wants to and can reasonably click that "DAG-CBOR" link and come back 10 minutes later knowing what IPLD is, if they don't already... because if _I_ am confused, who is your target reader 😏